cpp/src/gandiva/precompiled/string_ops.cc

```diff
@@ -131,6 +133,16 @@ bool is_substr_utf8_utf8(const char* data, int32_t data_len, const char* substr,
   return false;
 }
 
+FORCE_INLINE
+const char* regexp_replace(const char* regexp, const char* str, const char* replace_string) {
+    std::string s = str;
+    std::regex r (regexp);
```

Comment:
Please use RE2 instead of std::regex. Also use a holder to prevent compiling the pattern on each call. See like_holder.h

Reply:
I meant for to not use this function and use a function holder instead. Looks like you just added a holder and not using that and instead still calling this function